### PR TITLE
Improve French Locale

### PR DESCRIPTION
**Pull request by Lytzeer**
Hello, I started translating texts into French that were only in English. I am still continuing to translate the rest of the texts into French but here is already a part that I find important

### CODE REVIEW

**Comment by Lytzeer** [comment]
I also set the text in English by default because I don't know other languages

File: src/locales/de/egg.ts [hunk 1]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "Du hast nicht genug Ei-Gutscheine!",
   "tooManyEggs": "Du hast schon zu viele Eier!",
   "pull": "Pull",
-  "pulls": "Pulls"
+  "pulls": "Pulls",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/de/egg.ts:21]: Not in French here

File: src/locales/es/egg.ts [hunk 2]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "You don't have enough vouchers!",
   "tooManyEggs": "You have too many eggs!",
   "pull": "Pull",
-  "pulls": "Pulls"
+  "pulls": "Pulls",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/es/egg.ts:21]: Not in French here

File: src/locales/it/egg.ts [hunk 3]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "Non hai abbastanza Biglietti!",
   "tooManyEggs": "Hai troppe Uova!",
   "pull": "Tiro",
-  "pulls": "Tiri"
+  "pulls": "Tiri",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/it/egg.ts:21]: Not in French here

File: src/locales/pt_BR/egg.ts [hunk 4]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "Você não tem vouchers suficientes!",
   "tooManyEggs": "Você já tem muitos ovos!",
   "pull": "Prêmio",
-  "pulls": "Prêmios"
+  "pulls": "Prêmios",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/pt_BR/egg.ts:21]: Not in French here

File: src/locales/zh_CN/egg.ts [hunk 5]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "你没有足够的兑换券！",
   "tooManyEggs": "你的蛋太多啦！",
   "pull": "次",
-  "pulls": "次"
+  "pulls": "次",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/zh_CN/egg.ts:21]: Not in French here

File: src/locales/zh_TW/egg.ts [hunk 6]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "你沒有足夠的兌換券！",
   "tooManyEggs": "你的蛋太多啦！",
   "pull": "抽",
-  "pulls": "抽"
+  "pulls": "抽",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/zh_TW/egg.ts:21]: Not in French here

**Review by Dakurei — COMMENTED** [review]
I suppose there are a whole bunch of other problems, but that's quite a lot 😅

Potentially, there may be PRs that already take care of that, but I haven't checked, so let's see before going any further

File: src/locales/fr/egg.ts [hunk 7]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "Vous n’avez pas assez de coupons !",
   "tooManyEggs": "Vous avez trop d’Œufs !",
   "pull": "Tirage",
-  "pulls": "Tirages"
+  "pulls": "Tirages",
+  "eggHatched": "{{pokemonName}} a éclos de l’Œuf !"
```
Comment (Dakurei) [inline src/locales/fr/egg.ts:21]:
```suggestion
  "eggHatched": "Un {{pokemonName}} vient de sortir de l’Œuf !"
```

Wrong sentence in French
pokemon-ecarlate-violet-ouisticram-tortipouss-tiplouf-illu

Comment (Lytzeer) [reply src/locales/fr/egg.ts:21]: Yeah your sentence for the egg is better !

**Comment by Lytzeer** [comment]
I'm going to edit the egg sentence to put it in English because I actually didn't put the right one

**Comment by Lytzeer** [comment]
now it should be better

File: src/data/ability.ts [hunk 8]
```diff
@@ -1248,7 +1248,7 @@ export class PostDefendStealHeldItemAbAttr extends PostDefendAbAttr {
           const stolenItem = heldItems[pokemon.randSeedInt(heldItems.length)];
           pokemon.scene.tryTransferHeldItemModifier(stolenItem, pokemon, false, false).then(success => {
             if (success) {
-              pokemon.scene.queueMessage(getPokemonMessage(pokemon, ` stole\n${attacker.name}'s ${stolenItem.type.name}!`));
+              pokemon.scene.queueMessage(getPokemonMessage(pokemon, ` stole\n${attacker.name}'s ${stolenItem.type.name}! amodif`));
```
Comment (CodeTappert) [inline src/data/ability.ts:1251]: Could you pleaes explain this change.

File: src/locales/de/config.ts [hunk 9]
```diff
@@ -22,7 +23,7 @@ import { battleMessageUiHandler } from "./battle-message-ui-handler";
 import { berry } from "./berry";
 import { voucher } from "./voucher";
 
-export const deConfig = {
+export const frConfig = {
```
Comment (CodeTappert) [inline src/locales/de/config.ts:26]: You should not change this! Please change it back to deConfig

File: src/locales/de/egg.ts [hunk 10]
```diff
@@ -17,5 +17,6 @@ export const egg: SimpleTranslationEntries = {
   "notEnoughVouchers": "Du hast nicht genug Ei-Gutscheine!",
   "tooManyEggs": "Du hast schon zu viele Eier!",
   "pull": "Pull",
-  "pulls": "Pulls"
+  "pulls": "Pulls",
+  "eggHatched": "{{pokemonName}} hatched from the egg!"
```
Comment (CodeTappert) [inline src/locales/de/egg.ts:21]: `Ein {{pokemonName}} ist aus dem Ei geschlüpft!`

- Source Pokemon S/V

File: src/locales/es/config.ts [hunk 11]
```diff
@@ -22,7 +23,7 @@ import { battleMessageUiHandler } from "./battle-message-ui-handler";
 import { berry } from "./berry";
 import { voucher } from "./voucher";
 
-export const esConfig = {
+export const frConfig = {
```
Comment (CodeTappert) [inline src/locales/es/config.ts:26]: Also change this back to esConfig

File: src/locales/it/config.ts [hunk 12]
```diff
@@ -22,7 +23,7 @@ import { battleMessageUiHandler } from "./battle-message-ui-handler";
 import { berry } from "./berry";
 import { voucher } from "./voucher";
 
-export const itConfig = {
+export const frConfig = {
```
Comment (CodeTappert) [inline src/locales/it/config.ts:26]: change this back to itConfig

File: src/locales/de/phases.ts [hunk 13]
```diff
@@ -0,0 +1,15 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const phases: SimpleTranslationEntries = {
```
Comment (CodeTappert) [inline src/locales/de/phases.ts:3]:
```
 "berryNervous": "{{pokemonName}} ist zu\nnervös um Beeren zu essen!",
  "restoredHp": "Die KP von{{pokemonName}}\n wurden wiederhergestellt",
  "moveDisabled": "{{moveName}} ist blockiert!",
  "moveUsed": "{{pokemonName}} setzt\n{{moveName}} ein!",
  "attackMissed": "Die Attacke von {{pokemonName}}\n hat verfehlt!",
  "fainted": "{{pokemonName}} wurde besiegt!",
  "rewards": "Du erhältst\n{{rewardsName}}!",
  "gamemodeReward": "{{name}} beendete {{gamemodeName}}-Modus zum ersten Mal!\nDu erhältst {{rewardsName}}!",
  "retry": ""Willst du diesen Kampf wiederholen?", 
  "unlocked": "{{unlockableName}}\nwurde freigeschaltet.",
  "fullHp": "Die KP von {{pokemonName}}\n sind voll!",
```

**Comment by bennybroseph** [comment]
I'm honestly afraid of this PR considering all the error with either a refactor or copy/paste.

**Comment by CodeTappert** [comment]
> I'm honestly afraid of this PR considering all the errors with either a refactor or copy/paste. Please remake this with the correct changes.

RIP me playing through Pokemon Oras on 4x speed to get the german text xD

**Pull request closed without merging**